src/python/qeopenfermion/__init__.py

```diff
@@ -1,2 +1,9 @@
 from ._io import *
-from ._utils import *
\ No newline at end of file
+from ._utils import *
+import warnings
+
+warnings.warn(
+    """qe-openfermion will be fully deprecated \
+starting 2/1/2021. As of 12/15/2020, all functionality and future development \
+related to the qe-openfermion will be contained within zquantum.core.openfermion.""",
+)
```

Comment:
This adds a `UserWarning`. I think it would be better if we passed a second argument to `warn`, namely `DeprecationWarning`.

As a side note, you don't need to use multiline strings with line continuation, as placing multiple strings next to each other concatenates them. The above warning is equivalent to:
```python
warnings.warn(
    "qe-openfermion will be fully deprecated "
    "starting 2/1/2021. As of 12/15/2020, all functionality and future development "
    "related to the qe-openfermion will be contained within zquantum.core.openfermion.",
)
```